opdracht-tech-it-easy/src/main/java/nl/novi/opdrachttechiteasy/controllers/ExceptionController.java

```diff
@@ -0,0 +1,15 @@
+package nl.novi.opdrachttechiteasy.controllers;
+
+import nl.novi.opdrachttechiteasy.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+    @ControllerAdvice
+    public class ExceptionController {
+        @ExceptionHandler(RecordNotFoundException.class)
+        public ResponseEntity<String> handleRecordNotFoundException(RecordNotFoundException e) {
```

Comment:
Voor een uitbreiding op je exception handler, kijk even in de chat geschiedenis van de huiswerkles. Daar heb ik vorige week na afloop van de les een stuk code gedeeld om je exceptionhandling mee te verduidelijken (voor validatie errors)